### PR TITLE
fix: [TC-1225] - When opening an advisory to see its attributes, the display jumps to the top of the page

### PR DESCRIPTION
**Pull request by gildub**
Restore (as previously used with value of 0) fetch_interval to avoid user to browse page.

Fix https://issues.redhat.com/browse/TC-1225

### CODE REVIEW

**Comment by ctron** [comment]
IIRC @carlosthe19916 made that implementation. So I'll leave it to him to review as I don't recall the motivation for doing this.

**Review by carlosthe19916 — APPROVED** [review]
@gildub @ctron the reason for the continuous refresh is the fact that once a file is uploaded, that file is not intermediately available in the through the rest apis

-  User uploads a file (right now)
- It make take from seconds to minutes for the file to finally appear in the rest edpoint
- The continuous refresh is to avoid the user to manually refresh the page to see his file being available in the UI

We can merge this PR as long as we confirm the problem described above has been solved already. Otherwise, we should not merge this.

There was a specific requirement: "show the file uploaded in the UI without forcing the user to refresh, manually, the browser every now and then"

**Review by carlosthe19916 — COMMENTED** [review]
Could we confirm we do not have the "delay" issue anymore (files taking too much time to be indexed)?

**Comment by gildub** [comment]
@carlosthe19916, I see your point and effectively the initial issue doesn't seem to have been addressed.

In the meantime I realized the issue is actually occurring because the text field of the search text box is getting the focus.
So by turning off the auto-focus property on that field addresses the issue.

**Review by carlosthe19916 — APPROVED** [review]
@gildub Sorry for my late reply.
I tested your change locally and it works.
I don't think there is the need to make changes to the file `Cargo.lock` though

**Pull request merged**
